src/data/moonbeam/stellaswapLpPools.json

```diff
@@ -1,4 +1,23 @@
 [
+  {
+    "name": "stellaswap-xcaca-wglmr",
+    "address": "0xb83c50eA82c10AC598BA1ffc0c0296978c29c1e2",
+    "decimals": "1e18",
+    "poolId": 0,
```

Comment:
This doesn't seem like the correct pid?

Reply:
Nevermind I see you made it a dummy pool. This should actually go into a different folder. https://github.com/beefyfinance/beefy-api/blob/master/src/data/archive/oldLpPools.json

Reply:
Got it, I will move it to oldLpPools.json